go.mod

```diff
@@ -3,7 +3,9 @@ module github.com/dell/csi-vxflexos/v2
 // In order to run unit tests on Windows, you need a stubbed Windows implementation
 // of the gofsutil package. Use the following replace statements if necessary.
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.5
```

Comment:
Do we need to define a toolchain here?

Reply:
I am not sure on this, I'd have to defer this to Harshita.